src/praisonai-code/praisonai_code/cli/commands/sandbox.py

```
@@ -5,15 +5,61 @@
+    from praisonai_code._wrapper_bridge import import_wrapper_module
+
+    return import_wrapper_module("praisonai.cli.features.sandbox_cli").SandboxHandler()
+
+
+@app.command("run")
+def sandbox_run(
+    code: Optional[str] = typer.Option(None, "--code", "-c", help="Code to execute"),
+    file: Optional[str] = typer.Option(None, "--file", "-f", help="File to execute"),
+    sandbox_type: _SandboxType = typer.Option(
+        "subprocess", "--type", "-t", help="Sandbox backend (subprocess or docker)"
+    ),
```

Ruff (0.15.21)

[warning] Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)